catalog/security/banner-marking/src/main/java/org/codice/alliance/security/banner/marking/AeaMarking.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.security.banner.marking;
+
+import com.google.common.collect.ImmutableList;
+import java.io.Serializable;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import org.apache.commons.collections.CollectionUtils;
+
+/** Atomic Energy Act marking parser. */
+public class AeaMarking implements Serializable {
+  private static final Pattern SIGMAS_PATTERN = Pattern.compile(" ");
+  private AeaType type;
+
+  private boolean cnwdi;
```

Comment:
nit : we may want to expand this acronym to what it means in order to help other developers in the future

Reply:
Fixed.